drivers/soundwire/intel.c

```diff
@@ -1769,7 +1754,12 @@ static int intel_resume_runtime(struct device *dev)
 			return ret;
 		}
 	} else if (!clock_stop_quirks) {
-		/* FIXME: is this the right sequence ? */
+		ret = intel_init(sdw);
```

Comment:
in this function,  we will found clock stop = 1, so we don't need to initialize master.